tests/test_readers_s3.py

```diff
@@ -270,6 +272,7 @@ def test_get_batch(self):
         self.assertEqual(batch, expected_batch)
         shutil.rmtree(reader.tmp_folder, ignore_errors=True)
 
+    @freeze_time('2010-01-01')
```

Comment:
I added this as some tests failed locally due to a time change. Maybe I should do another PR for it to master branch directly.